bash_mapbox_route_visualizer.sh

```diff
@@ -191,19 +183,20 @@ UPLOAD_RESPONSE=$(curl -X PUT -T route.geojson \
   -H "x-amz-acl: public-read" \
   "https://${BUCKET}.s3.amazonaws.com/${KEY}")
 
-echo -e "  \033[0;32m✓\033[0m S3 upload completed successfully!"
-
+# shellcheck disable=SC2181
```

Comment:
checking the exit code of the previous command is fragile, but good enough for this